content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -65,8 +67,9 @@ This includes:
 1. **Permitted x509 extensions**: including IP subjectAltNames, DNS subjectAltNames, Email subjectAltNames, URI subjectAltNames etc, and behavior when a disallowed extension is requested.
 1. **Permitted key usages / extended key usages**: any restrictions on and behavior when usages different than the signer-determined usages are specified in the CSR.
 1. **Expiration/certificate lifetime**: whether it is fixed by the signer, configurable by the admin, determined by the CSR object etc and behavior if an expiration different than the signer-determined expiration is specified in the CSR.
-1. **CA bit allowed/disallowed**: and behavior if a CSR contains a request a for a CA cert when the signer does not permit it.
-1. **optional**: Information about the meaning of additional `CERTIFICATE` PEM blocks in `status.certificate`, if different from the standard behavior of treating the additional certificates as intermediates, and presenting them in TLS handshakes.
+1. **CA bit allowed/disallowed**: and behavior if a CSR contains a request a for a CA certificate when the signer does not permit it.
+
+Commonly, the `status.certificate` field contains a single PEM-encoded X.509 certificate once the CSR is approved and the certificate is issued. Some signers store multiple certificates into the `status.certificate` field. In that case, the documentation for the signer should specify the meaning of additional certificates; for example, this might be certificate plus intermediates to be presented during TLS handshakes.
```

Comment:
this should convey that the default meaning of additional certificates is that they are an intermediate chain

Reply:
To document this, I need a question answered: If there is more than one certificate, what's the convention for ordering?

For example:
- The first is the server's / clients certificate and the others chain onwards towards the root?
- The first is the intermediate certificate signed by the root,  then the next intermediate CA, and so on. The last in the list is the server's / client's certificate.
- The certificates are in any order; the TLS implementation should solve and find a valid chain then present that to a peer?
- something else?